x/ecocredit/client/tx.go

```diff
@@ -219,43 +223,31 @@ Required Flags:
 }
 
 func TxCreateBatchCmd() *cobra.Command {
-	var (
-		startDate = time.Unix(10000, 10000).UTC()
-		endDate   = time.Unix(10000, 10050).UTC()
-	)
-	createExampleBatchJSON, err := json.MarshalIndent(
-		ecocredit.MsgCreateBatch{
-			// Leave issuer empty, because we'll use --from flag
-			Issuer:  "",
-			ClassId: "1BX53GF",
-			Issuance: []*ecocredit.MsgCreateBatch_BatchIssuance{
-				{
-					Recipient:          "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw",
-					TradableAmount:     "1000",
-					RetiredAmount:      "15",
-					RetirementLocation: "ST-UVW XY Z12",
-				},
-			},
-			Metadata:        []byte{0x1, 0x2},
-			StartDate:       &startDate,
-			EndDate:         &endDate,
-			ProjectLocation: "AB-CDE FG1 345",
-		},
-		"                              ",
-		"    ",
-	)
-	if err != nil {
-		panic("Couldn't marshal MsgCreateBatch to JSON")
-	}
+
 	return txflags(&cobra.Command{
 		Use:   "create-batch [msg-create-batch-json-file]",
 		Short: "Issues a new credit batch",
 		Long: fmt.Sprintf(`Issues a new credit batch.
 
 Parameters:
   msg-create-batch-json-file: Path to a file containing a JSON object
-			      representing MsgCreateBatch. The JSON has format:
-                              %s`, createExampleBatchJSON),
+                              representing MsgCreateBatch. The JSON has format:
+                              {
+                                "class_id"": "C01",
+                                "issuance": [
+                                  {
+                                    "recipient":           "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw",
+                                    "tradable_amount":     "1000",
+                                    "retired_amount":      "15",
+                                    "retirement_location": "ST-UVW XY Z12",
+                                  },
+                                ],
+                                "metadata":         "AQI=",
+                                "start_date":       "1990-01-01",
+                                "end_date":         "1995-10-31",
+                                "project_location": "AB-CDE FG1 345",
+                              }
```

Comment:
the `Long` help text argument doesn't have access to the `*cobra.Command`, so its not possible to retrieve the sdkcontext and the JSON marshaler...  So I hardcoded an example JSON help text here.